app/models/patient.rb

```diff
@@ -738,6 +738,25 @@ def self.order_by_risk(asc: true)
     order(Arel.sql(asc ? order_by : order_by_rev))
   end
 
+  # Order individuals based if they are or are not flagged for follow-up
+  def self.order_by_follow_up_flag(asc: true)
+    order_by = <<~SQL
+      CASE
+      WHEN follow_up_reason IS NOT NULL THEN 0
+      WHEN follow_up_reason IS NULL THEN 1
+      END
+    SQL
+
+    order_by_rev = <<~SQL
```

Comment:
so I think you only want to use this one for both cases (if you look at the example above, the value when NULL is always the highest which is how NULL values always get sorted to the back)

Reply:
got it, in that case, I think we can just do this in `patient_query_helper`
```
when 'flagged_for_follow_up'
            patients = patients.order(Arel.sql('CASE WHEN follow_up_reason IS NULL THEN 1 ELSE 0 END, follow_up_reason ' + dir), id: dir)
```

Reply:
That query actually results in the flagged monitorees appearing at the top of the list regardless of sorting direction (I believe I used a similar query when I first implemented this before Sarah's feedback).

Reply:
alright, I haven't been able to think of a better solution for now so I'll resolve this